### ISSUE TITLE
Compression support?

### ISSUE THREAD

**Issue by Schachte** [mostafa/xk6-kafka#14]
We're getting the following error when pulling data from our compressed topic. Any chance there is a way to specify that we are using snappy compression? 

```
There was an error fetching messages: the compression code is invalid or its codec has not been importedINFO[0012]                                               source=console
There was an error fetching messages: the compression code is invalid or its codec has not been importedINFO[0012]                                               source=console
There was an error fetching messages: the compression code is invalid or its codec has not been importedINFO[0013]                                               source=console
There was an error fetching messages: the compression code is invalid or its codec has not been importedINFO[0014]                                               source=console
There was an error fetching messages: the compression code is invalid or its codec has not been importedINFO[0015]                                               source=console
There was an error fetching messages: the compression code is invalid or its codec has not been importedINFO[0015]                                               source=console
There was an error fetching messages: the compression code is invalid or its codec has not been importedINFO[0015]                                               source=console
```

**Comment by mostafa** [comment]
@Schachte 
Snappy is not supported yet. Would you like to add it?

**Comment by mostafa** [comment]
@Schachte 

I've made a PR that adds support for compression to `produce` and `createTopic` functions. Please test it and report any issues.

**Comment by mostafa** [comment]
I've tested the feature and merged the PR #18. Feel free to open a new issue if there's any issues.
